PLUGINS.md

```diff
@@ -0,0 +1,35 @@
+# Plugins
+
+The iXBRL Viewer provides a simple plugin framework, to enable projects to
+build a viewers with additional functionality.
+
+A plugin is a class with one or more of the following methods:
+
+* `extendDisplayOptionsMenu(menu)` - provides a menu object that can be modified to add additional options.
+* `preProcessiXBRL(body, docIndex)` - called once with the `<body>` DOM node for each document in the document set.
+* `updateViewerStyleElements(styleElts)` - called on initialisation with a jQuery object containing the `<style>` element from each document in the document set, allow additional styles to be registered.
```

Comment:
initialisation -> initialization

Reply:
American English vs British English ;-).

Reply:
Interesting!

Reply:
I try to stick to US spellings in code for consistency, but sometimes slip into my native tongue in documentation :-)
Changed in ef0066a95576fe1953780a729fe19b68f7a2db30